packages/api/test/car.spec.js

```diff
@@ -44,10 +55,14 @@ describe('POST /car', () => {
     assert(cid, 'Server response payload has `cid` property')
     assert.strictEqual(cid, expectedCid, 'Server responded with expected CID')
 
-    const statusRes = await fetch(new URL(`status/${cid}`, endpoint))
-    const status = await statusRes.json()
-    const pinInfo = status.pins.find(pin => PIN_OK_STATUS.includes(pin.status))
-    assert(pinInfo, `status is one of ${PIN_OK_STATUS}`)
+    const pinInfo = await retry(async () => {
```

Comment:
This is now done in the `waitUntil` so we retry until we get the result we want.

Reply:
It would be great to consider changing to Ava here and rely on Miniflare lib to wait for this kind of things

Reply:
I agree, but seperate from this PR!